SaurabhM  Before:  Missing menu button in mobile view and contact page  After:  Added menu button in mobile view, improved responsiveness and Created Contact page with responsive

saurabh142003 opened the pull request:
### Related Issue
- Closes: #<issue number>

---

### Description

<!-- Briefly describe the changes in this PR -->

---

### How Has This Been Tested?

<!-- Briefly describe how you tested the changes -->

---

### Screenshots (if applicable)

<!-- Add screenshots to show visual changes, if applicable -->

---

### Type of Change

<!-- Check the type of change this PR introduces -->

- [ ] Bug fix
- [ ] New feature
- [ ] Code style update
- [ ] Breaking change
- [ ] Documentation update

mehul-m-prajapati commented:
@saurabh142003 : please fill description

saurabh142003 commented:
> @saurabh142003 : please fill description

i can't able to edit it i think i don't have permission to change the description

saurabh142003 commented:
please guide me if possible

saurabh142003 commented:
> @saurabh142003 : please fill description

Done

mehul-m-prajapati commented:
@saurabh142003 : please reduce the size of font of navbar text => Dashboard, About, Contact

saurabh142003 commented:
> @saurabh142003 : please reduce the size of font of navbar text => Dashboard, About, Contact
Done

mehul-m-prajapati commented:
@saurabh142003 : where is html code of about page?

saurabh142003 commented:
> @saurabh142003 : where is html code of about page?

Done

mehul-m-prajapati commented:
@saurabh142003 : resolve conflicts and remove responsiveness changes.

Add only about, contact page changes

saurabh142003 commented:
> @saurabh142003 : resolve conflicts and remove responsiveness changes.
> 
> Add only about, contact page changes

So i need to remove the navbar responsiveness that i had added and what are the conflicts that i have to resolve?

saurabh142003 commented:
> @saurabh142003 : resolve conflicts and remove responsiveness changes.
> 
> Add only about, contact page changes

1. I have removed my responsiveness from the navbar
2.  Changed app.jsx file by changing the route path structure
3.  I have changed the anchor tag to link tag when it is mob- responsive because it was reloading the page in mobile responsive
![photo-collage png](https://github.com/user-attachments/assets/9541bd73-2902-418d-b458-f0221b52143d)

# Query :  Do i have to change the folder structure by removing the about and contact page from component folder and add it to pages folder?

mehul-m-prajapati commented:
@saurabh142003 : open a pull request with a branch. Dont commit your changes directly on main branch

You need to pull latest changes and then do yours

saurabh142003 commented:
> @saurabh142003 : open a pull request with a branch. Dont commit your changes directly on main branch
> 
> You need to pull latest changes and then do yours

what shall i do? the contact page and about page has already been made.

saurabh142003 commented:
> @saurabh142003 : open a pull request with a branch. Dont commit your changes directly on main branch
> 
> You need to pull latest changes and then do yours

what shall i do? the contact page and about page has already been made.
![Screenshot 2024-11-18 234416](https://github.com/user-attachments/assets/5d498796-06d3-4df3-8b29-1dfee58f72d5)
![Screenshot 2024-11-18 234429](https://github.com/user-attachments/assets/b32e9896-d33d-4672-a7e8-5b72066a4bca)

mehul-m-prajapati commented:
@saurabh142003 - please work on some other issue

saurabh142003 commented:
> @saurabh142003 - please work on some other issue

No i'm not gonna do thank you for wasting my time!
